backup.py

```diff
@@ -82,15 +81,18 @@ def stream_to_s3(self, url, remote_filename):
             connect = boto.connect_s3()
         else:
             connect = boto.connect_s3(
-                aws_access_key_id=self.config['UPLOAD_TO_S3']['AWS_ACCESS_KEY'], 
-                aws_secret_access_key=self.config['UPLOAD_TO_S3']['AWS_SECRET_KEY']
+                host=self.config['UPLOAD_TO_S3']['AWS_ENDPOINT_URL'],
+                aws_access_key_id=self.config['UPLOAD_TO_S3']['AWS_ACCESS_KEY'],
+                aws_secret_access_key=self.config['UPLOAD_TO_S3']['AWS_SECRET_KEY'],
+                is_secure=self.config['UPLOAD_TO_S3']['AWS_IS_SECURE']
                 )
+            connect.auth_region_name = self.config['UPLOAD_TO_S3']['AWS_REGION']
 
         bucket = connect.get_bucket(self.config['UPLOAD_TO_S3']['S3_BUCKET'])
         r = self.session.get(url, stream=True)
         if r.status_code == 200:
             k = Key(bucket)
-            k.key = remote_filename
+            k.key = "%s%s" % (self.config['UPLOAD_TO_S3']['S3_DIR'], remote_filename)
```

Comment:
The rest of the printting in this code is wirtten with `format`, so I think it will be easier to read the code if we will keep using one standart.

Reply:
fixed